src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -668,6 +667,16 @@ def test_process_declarations(f2008_parser):
     assert isinstance(fake_parent.symbol_table.lookup("i5").constant_value,
                       BinaryOperation)
 
+    # Initialisation with constant expresions with symbols
```

Comment:
Initialise with a constant expression (`1`) and with a symbol (`val1`).

Reply:
Done